src/app/api/routes/devices.py

```diff
@@ -35,7 +35,10 @@ async def update_device(payload: DeviceIn, device_id: int = Path(..., gt=0)):
 
 @router.delete("/{device_id}/", response_model=DeviceOut)
 async def delete_device(device_id: int = Path(..., gt=0), _=Security(get_current_user, scopes=["admin"])):
```

Comment:
Can we do a "db" transaction like what JB suggested in one o his reviews?

Reply:
Considering the PR is still hanging and I don't know how to use them as of now, I didn't consider this for this PR. But I still have one last thing to check with this PR (some troubles in the unittests)